src/Modules/StatCan.OrchardCore.Assessment/Views/Layout.liquid

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```

Comment:
You should not place layout files in modules, the layout is controlled by the selected theme